.travis.yml

```diff
@@ -82,12 +87,17 @@ after_success:
  - ccache -s
  - test "${TRAVIS_BUILD_STAGE_NAME}" = "Test" || travis_terminate 0
  - if [[ ${COVERAGE} ]]; then
-     pushd build && wget -O codecov https://codecov.io/bash && chmod +x codecov &&
+     pushd build &&
      if [[ ${CC} = clang ]]; then
-       ./codecov -x "llvm-cov gcov" -F "${CC}";
+       codecov -x "llvm-cov gcov" -F "${CC}";
```

Comment:
What is different about coverage with LLVM?

Reply:
For `clang` you need to use `llvm-cov gcov` instead of `gcov`.